src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -35,6 +35,11 @@ public class TwilioRestClient : ITwilioRestClient
         /// </summary>
         public string Region { get; }
```

Comment:
Should this also be `{ get; set; }` like Edge?

Reply:
The idea is since region is tied to auth, so you shouldn't be updating it standalone. But edge is fine.

Reply:
Ah, right.